projects/igniteui-angular/src/lib/data-operations/filtering-strategy.ts

```diff
@@ -139,3 +140,23 @@ export class FormattedValuesFilteringStrategy extends FilteringStrategy {
         return value;
     }
 }
+
+export class PivotFilteringStrategy extends FilteringStrategy {
```

Comment:
This looks very similar to the existing DimensionValuesFilteringStrategy. Can't we re-use that one?